benchmark/bench-report/bin/bench-runner.sh

```diff
@@ -18,6 +18,7 @@ print_help () {
   echo "       [--benchmarks <"bench1 bench2 ..." | help>]"
   echo "       [--prefix <benchmark name prefix to match>"
   echo "       [--fields <"field1 field2 ..." | help>]"
+  echo "       [--failure-field]"
```

Comment:
We can call it `--fail-on`.

Reply:
We probably don't need this. It should fail on all the fields. We can filter fields accordingly though.